client.go

```diff
@@ -27,10 +29,24 @@ var (
 	ErrBadRequest = errors.New("error: bad request")
 )
 
+// Error API Response. Contains the error message as well as the type of error
+type Error struct {
```

Comment:
This Error seems to only occur in the CRUDResponse. Should this be in the user package?

Reply:
This is the general error response from TV which is why I tossed it in the client. Happy to change if needed

Reply:
I see. I think this is good :+1: I might need to check the error response in the  Document Search too to accommodate this error.